packages/terra-clinical-item-display/src/ItemDisplay.jsx

```diff
@@ -43,6 +44,16 @@ const propTypes = {
    * One of `'center'`, `'top'`, `'inline'`.
    */
   iconAlignment: PropTypes.oneOf(['center', 'top', 'inline']),
+  /**
+   * The meaning of the text based on styling, used by screen readers.
+   * Changing `meaning` will not visually change the style of the content.
+   */
+  meaning: PropTypes.string,
```

Comment:
`meaning` is a bit vague. Can we update this to something like `textMeaning` (or any other suitable alternative)?

Reply:
> `meaning` is a bit vague. Can we update this to something like `textMeaning` (or any other suitable alternative)?

Good catch, I would even expand that to read `textStyleMeaning`, as my understanding is we are adding this prop to explain the visual styling of the text rather than text itself to the assistive technology